user-operator/src/main/java/io/strimzi/operator/user/UserOperatorConfig.java

```diff
@@ -128,6 +129,11 @@ public class UserOperatorConfig {
      */
     public static final ConfigParameter<List<String>> MAINTENANCE_TIME_WINDOWS = new ConfigParameter<>("STRIMZI_MAINTENANCE_TIME_WINDOWS", SEMICOLON_SEPARATED_LIST, "", CONFIG_VALUES);
 
+    /**
+     * Configuration string with feature gates settings
+     */
+    public static final ConfigParameter<FeatureGates> FEATURE_GATES = new ConfigParameter<>("STRIMZI_FEATURE_GATES", parseFeatureGates(), "", CONFIG_VALUES);
```

Comment:
This looks good. But I guess you also need to pass it to the user operator based on the setting in the cluster operator.